### PR TITLE
Added copy constructor

### PR DESCRIPTION
**Pull request by axla-io**
I added a test, but it can be omitted.

### CODE REVIEW

File: src/FastAlmostBandedMatrices.jl [hunk 1]
```diff
@@ -127,6 +127,7 @@ end
 @inline Base.size(A::AlmostBandedMatrix) = size(A.bands)
 @inline Base.eltype(::AlmostBandedMatrix{T}) where {T} = T
 @inline Base.IndexStyle(::Type{<:AlmostBandedMatrix}) = IndexCartesian()
+@inline Base.copy(A::AlmostBandedMatrix) = AlmostBandedMatrix(bandpart(A), fillpart(A))
```
Comment (avik-pal) [inline src/FastAlmostBandedMatrices.jl:130]: This is not correct, you need to call copy on them.

**Review by avik-pal — CHANGES_REQUESTED** [review]

**Comment by avik-pal** [comment]
Also run the formatter to make CI happy

File: src/FastAlmostBandedMatrices.jl [hunk 2]
```diff
@@ -3,13 +3,13 @@ module FastAlmostBandedMatrices
 import PrecompileTools: @recompile_invalidations, @setup_workload, @compile_workload
 
 @recompile_invalidations begin
-    using ArrayInterface, ArrayLayouts, BandedMatrices, ConcreteStructs, LazyArrays,
-        LinearAlgebra, MatrixFactorizations, Reexport
+	using ArrayInterface, ArrayLayouts, BandedMatrices, ConcreteStructs, LazyArrays,
+		LinearAlgebra, MatrixFactorizations, Reexport
```
Comment (avik-pal) [inline src/FastAlmostBandedMatrices.jl:7]: You editor changed all the spaces into tabs

**Review by avik-pal — CHANGES_REQUESTED** [review]

File: src/FastAlmostBandedMatrices.jl (hunk 2 above)
Comment (axla-io) [reply src/FastAlmostBandedMatrices.jl:7]: :((( Maybe my formatter is badly setup...

File: test/runtests.jl [hunk 3]
```diff
@@ -15,6 +15,7 @@ using SafeTestsets, Test
 
         @test almostbandwidths(A) == (2, 1)
         @test almostbandedrank(A) == 2
+        @test copy(A) == A
```
[the remark this answers is not in the document]
Comment (axla-io) [reply test/runtests.jl:18]: Alright!

**Pull request closed without merging**